app/main/helpers/validation.py

```diff
@@ -218,7 +226,7 @@ def get_required_fields(self):
 
 
 class G8Validator(DOSValidator):
-    pass
+    number_string_fields = [('dunsNumber', 9)]
```

Comment:
Yep as far as I know DUNS number is the only thing we can specify a format for - company number, etc.  I don't think we can, as can be `N/A`.